contrib/src/main/java/com/datatorrent/contrib/schema/formatter/JsonFormatter.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (C) 2015 DataTorrent, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datatorrent.contrib.schema.formatter;
+
+import java.text.SimpleDateFormat;
+
+import org.codehaus.jackson.map.ObjectMapper;
+import org.codehaus.jackson.map.ObjectWriter;
+import org.codehaus.jackson.map.SerializationConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.datatorrent.api.Context;
+
+/**
+ * Operator that converts POJO to JSON string
+ * <br><b>Properties</b>
+ *  <br><b>dateFormat</b>: date format
+ *  e.g dd/MM/yyyy
+ * @displayName JsonFormatter
+ * @category Formatter 
+ * @tags  pojo json formatter
+ */
+public class JsonFormatter extends Formatter<String>
+{
+
+  private transient ObjectWriter writer;
+  protected String dateFormat;
+  
+  @Override
+  public void activate(Context context)
+  {
+    try {
+      ObjectMapper mapper = new ObjectMapper();
+      if (dateFormat != null ) {
+        mapper.setDateFormat(new SimpleDateFormat(dateFormat));
+      }
+      writer = mapper.writerWithType(clazz);
+      mapper.configure(SerializationConfig.Feature.AUTO_DETECT_FIELDS, true);
+      mapper.configure(SerializationConfig.Feature.AUTO_DETECT_GETTERS, true);
+      mapper.configure(SerializationConfig.Feature.AUTO_DETECT_IS_GETTERS, true);
+    } 
+    catch (Throwable e) {
+      throw new RuntimeException("Unable find provided class");
+    }
+  }
+
+  @Override
+  public void deactivate()
+  {
+    // TODO Auto-generated method stub
+    
+  }
+
+  @Override
+  public String convert(Object tuple)
+  {
+    try {
+      return writer.writeValueAsString(tuple);
+    }
+    catch (Exception ex) {
+      logger.error("error while deserializing tuple",ex.getMessage());
```

Comment:
Hi Thomas, the requirement was to  emit tuples that could not be parsed on error port. Hence, i chose to not propogate exception but rather to log them and emit the tuple on error port. Please guide if cases like these should be handled in a different way